src/ResXManager.View/Tools/ResXFileWatcher.cs

```diff
@@ -65,6 +65,13 @@ protected override void OnFilesChanged()
                 if (language.HasChanges)
                     continue;
 
+                var config = _resourceManager.Configuration;
+                var sortFileContentOnSave = config.SortFileContentOnSave;
+                var resxSortingComparison = config.ResXSortingComparison;
+                var duplicateKeyHandling = config.DuplicateKeyHandling;
+                if (sortFileContentOnSave)
+                    language.SortNodes(resxSortingComparison, duplicateKeyHandling);

```

Comment:
add braces if its more than a simple statement

Reply:
What about this?

Reply:
sorry , forgot it , fixed